mozdownload/cli.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from optparse import OptionParser, OptionGroup
+import os
+import pkg_resources
+import sys
+
+from . import errors
+from . import scraper
+
+
+version = pkg_resources.require("mozdownload")[0].version
+
+__doc__ = """
+Module to handle downloads for different types of archive.mozilla.org hosted \
```

Comment:
That's an old relict and hasn't been newly added.  Would you be ok when I fix that with my next PR? It's already fixed there locally.